frontend/src/metabase/lib/entity-id/hooks/use-validated-entity-id.ts

```diff
@@ -54,7 +54,7 @@ export const useValidatedEntityId = <
   );
 
   return useMemo(() => {
-    if (!isEntityId) {
+    if (isNumber(id)) {
```

Comment:
some strings are also not an EntityId and we don't want to return yet, we should only return when the ID is number because there's nothing to resolve.